src/js/providers/tracks-mixin.js

```diff
@@ -385,6 +385,10 @@ define(['../utils/underscore',
         return providerName.indexOf('flash') === -1 && (utils.isChrome() || utils.isIOS() || utils.isSafari());
     }
 
+    function _kindSupported(kind) {
```

Comment:
We should replace all of ```if (track.kind === 'subtitles' || track.kind === 'captions')``` with this function if we are creating this. Here is one place: https://github.com/jwplayer/jwplayer/blob/f8fef3d470d92509ac23dff16aa663b61ca1b8a9/src/js/providers/tracks-mixin.js#L69
Try to check for other places too.